frontend/viewer/package.json

```diff
@@ -31,83 +31,83 @@
     "build-storybook": "storybook build"
   },
   "devDependencies": {
-    "@argos-ci/playwright": "^5.0.4",
-    "@chromatic-com/storybook": "^4",
+    "@argos-ci/playwright": "^5.0.8",
+    "@chromatic-com/storybook": "^4.0.1",
     "@egoist/tailwindcss-icons": "^1.9.0",
-    "@iconify-json/mdi": "^1.1.66",
-    "@lingui/cli": "^5.2.0",
-    "@lingui/format-json": "^5.2.0",
-    "@lingui/vite-plugin": "^5.2.0",
+    "@iconify-json/mdi": "^1.2.3",
+    "@lingui/cli": "^5.3.3",
+    "@lingui/format-json": "^5.3.3",
+    "@lingui/vite-plugin": "^5.3.3",
     "@mdi/js": "^7.4.47",
     "@playwright/test": "catalog:",
-    "@storybook/addon-a11y": "^9.0.8",
-    "@storybook/addon-docs": "^9.0.8",
-    "@storybook/addon-svelte-csf": "^5.0.3",
-    "@storybook/addon-vitest": "^9.0.8",
-    "@storybook/svelte-vite": "^9.0.8",
+    "@storybook/addon-a11y": "^9.0.18",
+    "@storybook/addon-docs": "^9.0.18",
+    "@storybook/addon-svelte-csf": "^5.0.7",
+    "@storybook/addon-vitest": "^9.0.18",
+    "@storybook/svelte-vite": "^9.0.18",
     "@stylistic/eslint-plugin": "catalog:",
     "@sveltejs/vite-plugin-svelte": "catalog:",
     "@tailwindcss/container-queries": "^0.1.1",
     "@tailwindcss/typography": "^0.5.16",
     "@tsconfig/svelte": "^5.0.4",
-    "@types/node": "^20",
-    "@typescript-eslint/eslint-plugin": "catalog:",
+    "@types/node": "^20.19.9",
     "@typescript-eslint/parser": "catalog:",
     "@vitest/browser": "catalog:",
     "@vitest/ui": "catalog:",
     "autoprefixer": "^10.4.20",
     "bits-ui": "2.8.11",
     "clsx": "^2.1.1",
     "eslint": "catalog:",
-    "eslint-plugin-storybook": "^9.0.8",
+    "eslint-plugin-storybook": "^9.0.18",
     "eslint-plugin-svelte": "catalog:",
-    "mode-watcher": "^1.0.7",
+    "mode-watcher": "^1.1.0",
     "paneforge": "1.0.0-next.4",
     "playwright": "catalog:",
-    "storybook": "^9.0.8",
+    "storybook": "^9.0.18",
     "svelte": "catalog:",
     "svelte-check": "catalog:",
     "svelte-eslint-parser": "catalog:",
     "svelte-sonner": "^1.0.5",
-    "tailwind-merge": "^3.0.2",
+    "tailwind-merge": "^3.3.1",
     "tailwind-variants": "^1.0.0",
     "tailwindcss": "catalog:",
     "tailwindcss-animate": "^1.0.7",
     "tslib": "catalog:",
-    "tw-colors": "^3.3.1",
+    "typescript-eslint": "catalog:",
+    "tw-colors": "^3.3.2",
     "typescript": "catalog:",
     "vaul-svelte": "1.0.0-next.6",
     "vite": "catalog:",
-    "vite-plugin-webfont-dl": "^3.10.4",
+    "vite-plugin-webfont-dl": "^3.11.1",
     "vitest": "catalog:",
     "vitest-browser-svelte": "^0.1.0"
   },
   "dependencies": {
     "@formatjs/intl-durationformat": "^0.7.4",
-    "@lingui/core": "^5.2.0",
+    "@lingui/core": "^5.3.3",
     "@microsoft/dotnet-js-interop": "^8.0.0",
-    "@microsoft/signalr": "^8.0.0",
-    "autoprefixer": "^10.4.19",
+    "@microsoft/signalr": "^8.0.7",
+    "autoprefixer": "^10.4.21",
```

Comment:
You just bumped `"autoprefixer"` from `"^10.4.19"` to `"^10.4.21"` here in the `dependencies`, but there's  `"autoprefixer": "^10.4.21",` in the `devDependencies` of `frontend/package.json`.